src/createDelete.js

```diff
@@ -54,9 +64,19 @@ const createTask = () => {
       tasks.length = 0;
       Array.prototype.push.apply(tasks, updatedTasks);
       createTask();
-      storage.saveTasks();
+      storage.saveTasks(tasks);
```

Comment:
- [ ] Your changes to tasks are being saved to local storage only when the checkbox is checked. Now when I clear all completed tasks, it doesn't persist to local storage.

- [ ] Kindly make sure that clear all completed does not remove all items from local storage. It should remove only the items that have been cleared. see video below for more details:

- https://www.loom.com/share/3de74bd9e311458594da2b4b2b966026

-  [ ] [OPTIONAL] Adding of books should also persist to local storage. I am making this optional because adding and removing of task is a To Do list: Add & Remove activity and milestone.